src/lib/dialog/dialog.ts

```diff
@@ -59,6 +68,20 @@ export class MdDialog {
   }
 
   /**
+   * Gets an observable that is notified when a dialog has been opened.
+   */
+  afterOpen(): Observable<any> {
```

Comment:
This is okay to just be a property rather than a method (in hindsight we shouldn't have made the `afterClosed` on the ref a method and will be changing it).